server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/MultiTermsIT.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.aggregations.bucket;
+
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.script.Script;
+import org.opensearch.script.ScriptType;
+import org.opensearch.search.aggregations.bucket.terms.BaseStringTermsTestCase;
+import org.opensearch.search.aggregations.bucket.terms.StringTermsIT;
+import org.opensearch.search.aggregations.bucket.terms.Terms;
+import org.opensearch.search.aggregations.support.MultiTermsValuesSourceConfig;
+import org.opensearch.search.aggregations.support.ValueType;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+import java.util.Collections;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.opensearch.search.aggregations.AggregationBuilders.multiTerms;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertSearchResponse;
+
+/**
+ * Extend {@link BaseStringTermsTestCase}.
+ */
+@OpenSearchIntegTestCase.SuiteScopeTestCase
+public class MultiTermsIT extends BaseStringTermsTestCase {
```

Comment:
Could we please add test cases for `bool` and `date` fields? I don't see those covered (please correct me if I am mistaken)

Reply:
date and boolean is covered in.
1. [UT](https://github.com/penghuo/OpenSearch/blob/multi-terms-agg/server/src/test/java/org/opensearch/search/aggregations/bucket/terms/MultiTermsAggregatorTests.java#L551).
2. [RestTest ](https://github.com/penghuo/OpenSearch/blob/multi-terms-agg/rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/370_multi_terms.yml#L131)

MultiTermsIT is used to test script.

Reply:
:+1: